hack/lib/golang.sh

```diff
@@ -24,6 +24,7 @@ readonly KUBE_GOPATH="${KUBE_OUTPUT}/go"
 # If you update this list, please also update build/release-tars/BUILD.
 kube::golang::server_targets() {
   local targets=(
+    cmd/gke-certificates-controller
```

Comment:
this still ends up in `kubernetes-server.tar.gz`. :( but at least there's not a docker tarfile, too.